app/__init__.py

```diff
@@ -2,18 +2,25 @@
 
 # Third party imports
 from flask_api import FlaskAPI, status, exceptions
+from flask_sqlalchemy import SQLAlchemy
 
 # local imports
 from config import app_config
 
+# initialize sql-alchemy
+db = SQLAlchemy()
+
 # Initialize the app
 app = FlaskAPI(__name__, instance_relative_config=True)
-app.secret_key = "string"
+# Load the config file
+app.config.from_object(app_config['development'])
+app.secret_key = "secret_string"
```

Comment:
consider getting  the secret key from an environment variable.

Reply:
sure